xsd/netex_framework/netex_reusableComponents/netex_facility_support.xsd

```diff
@@ -672,6 +672,11 @@ Rail transport, Roads and Road transport
 			<xsd:enumeration value="internet"/>
 			<xsd:enumeration value="postOffice"/>
 			<xsd:enumeration value="postBox"/>
+		<xsd:enumeration value="UsbPowerSocket">
```

Comment:
This should be reprocessed. I don't know why github did not do it.

Reply:
Likely because @TuThoThai did it in her own repo. Are you doing this on Windows or Linux?

Reply:
Trick is switching to Linux :-p and be able to run the github script yourself.